### DISCUSSION TITLE
How to sign a plugin without knowing the exact instance url?

### DISCUSSION THREAD

**Discussion by shuotli** [grafana/grafana#48914] [Plugins]
I see that `npx @grafana/toolkit plugin:sign --rootUrls https://example.com/grafana` works with signing the plugin on https://example.com/grafana. If we don’t know the exact url but only knows its pattern, (e.g. `*.grafana.com`), is there a way to sign it?

**Comment by daniellee** [comment]
Currently wildcards are not supported for the rootUrls parameter. There is an issue for this but it is not currently being worked on: https://github.com/grafana/grafana/issues/36355

It is something we are planning to support but don't have a timeline right now.

**Comment by timlevett** [comment]
It would be good to support multiple subdomains, so that if you had like `instance.env.comp.com` e.g. `**.x.com` it would be covered.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
